### ISSUE TITLE
Character bugged and can't play

### ISSUE THREAD

**Issue by Razielhell** [WarEmu/WarBugs#20511]
As Max requested. I can play on all of my other characters but not on my main.
![Warhammer Online - Age Of Reckoning Screenshot 2022 10 26 - 22 27 44 21](https://user-images.githubusercontent.com/16375224/198139008-cde9bd47-1a7f-4e52-8c70-9afdd36dc523.png)
![Warhammer Online - Age Of Reckoning Screenshot 2022 10 26 - 22 27 55 46](https://user-images.githubusercontent.com/16375224/198139031-f437cf5e-cdd7-42e3-8aff-d0828c8364df.png)

https://user-images.githubusercontent.com/16375224/198138963-82cbba31-44fa-4d44-8021-2088964b4175.mp4

**Comment by Lanissea** [comment]
I have the same problem

![image](https://user-images.githubusercontent.com/116764038/198139751-ed6a46d3-a872-4503-be3a-0c81c301295b.png)
![image](https://user-images.githubusercontent.com/116764038/198139841-60ea01f6-0a98-4341-a7d9-d24336ceb8ad.png)

**Comment by Lanissea** [comment]
apparently he considers me as connected ... the disconnection is not well produced

![image](https://user-images.githubusercontent.com/116764038/198144502-802df141-fa3f-4589-b6f8-6f5d0b003b7c.png)

**Comment by Lanissea** [comment]
news ?

**Comment by Razielhell** [comment]
Nope. Most likely a temporary fix would be next restart/patch but that's my personal thought.

**Comment by Lanissea** [comment]
I'm thinking about it too, but I hope not it's the only character I play :(

**Comment by Razielhell** [comment]
Yeap same for me it's my main character. We should wait for tomorrow patch. Hopefully even if it gets fixed with restart/patch this won't close and gets investigated for similar issues in the future.

**Comment by Lanissea** [comment]
yes, let's hope this solves the problem but it would be nice to know if it works on our problem.

**Comment by Razielhell** [comment]
ok my Warrior Priest cant login too now. I was playing a scenario > pressed leave > Took too long on scoreboard screen > I terminated the game > restarted the launcher > pressing PLAY on my WP does the same thing.

**Comment by kazzantip90** [comment]
Same problem with my WE Amidala since yesterday
![screenshot_021](https://user-images.githubusercontent.com/116856985/198479426-edd97873-1fd5-4507-861b-b438ab538c00.jpg)

**Comment by Lanissea** [comment]
I'm sorry to revive you here but a restart is planned today? a solution for our blocked characters? thanks in advance.

**Comment by Lanissea** [comment]
it works very well, thank you .

**Comment by MaxHayman** [comment]
Please post here if you are having this issue. Please give as much details as you can.

- How did you log off? disconnected or otherwise
- Where was you before? In a SC, dungone,
- What was you doing before?

It would be nice to find a pattern.

Afaik this started happening from Wednesdays patch and there was nothing related to anything which could cause this in any code changes since the last hotfix.

**Comment by kazzantip90** [comment]
Amidala
Its start with an error combat in TM. While was in stealth cant attack target. So I decide to relog.
I go to warcamp and logout from launcher as usual after 20 second timer. After that I cant log in back

**Comment by XmatXergon** [comment]
Have u guys tried to copy Warhammer folder, remove all addons and custom settings, and try log in from fresh Warhammer client ?

**Comment by Rhyshara** [comment]
**Hello, got the same problem on my character @Rhyshara !**
Can't login anymore.

    How did you log off? disconnected or otherwise
~ Logged off manually after doing some RvR in Dragonwake.

    Where was you before? In a SC, dungone,
~ Dragonwake, RvR zone. I think at the warcamp (destro).

    What was you doing before?
~ capping some BO with guildmembers.

It would be nice to find a pattern.

What's maybe important:
First attempt in loggin in -> 15-20secs loading screen -> back to char screen. Hitting play button again did not work -> needed to go back to char selection / restarting client.
Relocation to IC via GM did not help. Same problem.
When checking guild roster, the "last online" info seems to get updated by the login attempt.

**Comment by Razielhell** [comment]
> Have u guys tried to copy Warhammer folder, remove all addons and custom settings, and try log in from fresh Warhammer client ?

Yes..

**Comment by lilimx** [comment]
![image](https://user-images.githubusercontent.com/23266079/198889650-10270ab2-e857-4819-a414-979d26fe1f8a.png)

Character: Dathmoran
Where: KV - WC 
What: random disconnect

// It's a fresh case...so maybe interesting

**Comment by Rhyshara** [comment]
Can login again (@Rhyshara) ... did exactly nothing on my side... just gave it a try every day... back in :)

**Comment by Razielhell** [comment]
> Can login again (@Rhyshara) ... did exactly nothing on my side... just gave it a try every day... back in :)

Yes because of the restart.

**Comment by Rhyshara** [comment]
And can't login again.... Weekend Warfront Update went online in the meantime. If that helps :).

**Comment by Razielhell** [comment]
Happened again on my Witch Hunter right now. To reproduce: 
Finished Pyramid of Setra Scenario.
Pressed leave now button as usuall.
Didn't crash or anything. 
Animation of shield stopped at the bottom of scoreboard(I use it as an indicator that things are loading etc)
It pushed me back to main screen.
Pressed Quit to quit the game completelly. 
Restarted launher.
Same issue when pressing PLAY button on my WH.

**Comment by Arfaern** [comment]
Hello! I have the same problem. After an Sc I crashed to my loading screen and cant logg in this Character anymore. With other Characters its ok. Same was from another dude i had in Sc grp. (Bearbeitet)

After finishing "Pyramid of Settra" Sc.

**Comment by b00nation** [comment]
This happened to my once during a shining way fort, i was so pissed of getting killed that i used alt+F4. It ended on being stuck in loading screen in eataine. So i have the loading screen images deactivated, what is happening is that your character and abilities are already loaded just not the environment. Because of being able to see my actionbar i just used my book of binding and got ported to IC. I also to confirm it's bugged logged out+ in also closed the game and restarted it.

**Comment by Tesq** [comment]
i remember to had something similar, try start the game click login, then after 1sec close the whole thing; try a bunch of times see if it let you in at some point; also these kind of bug happened in past due GUILD bug. Try get kick from the guild.
